igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/service/JenkinsService.java

```diff
@@ -264,7 +271,8 @@ public void updateBuild(String jobName, Long buildNumber, UpdatedBuild updatedBu
 
```

Comment:
Retrofit2SyncCall.execute() is missing for jenkinsClient.submitDescription() at line 267